### ISSUE TITLE
Xception model training error 

### ISSUE THREAD

**Issue by nikkisingh111333** [deeplearning4j/deeplearning4j#9193]
**hey i m new in transfer learning and currently trying to use Xception model …heres what i m doing …**

_ZooModel zooModel = Xception.builder() 
.workspaceMode(WorkspaceMode.ENABLED) 
.cacheMode(CacheMode.DEVICE) 
.numClasses(2) .seed(randomSeed) .build(); 

zooModel.setInputShape(new int[][] {{3,150,150}});

 zooModel.init();

DL4JResources.setBaseDownloadURL("https://dl4jdata.blob.core.windows.net/"); 
ComputationGraph pretrainedNet = (ComputationGraph) zooModel.initPretrained(); 
FineTuneConfiguration fineTuneConf = new FineTuneConfiguration.Builder() .optimizationAlgo(OptimizationAlgorithm.STOCHASTIC_GRADIENT_DESCENT)
 .updater(new Nesterovs(5e-5)) .seed(123) .build(); 

ComputationGraph xception = new TransferLearning.GraphBuilder(pretrainedNet)
 .fineTuneConfiguration(fineTuneConf) .setFeatureExtractor("avg_pool") 
.removeVertexKeepConnections("predictions") 
.addLayer("predictions", new OutputLayer.Builder(LossFunctions.LossFunction.NEGATIVELOGLIKELIHOOD)
 .nOut(2) .weightInit(WeightInit.XAVIER) 
.activation(Activation.SOFTMAX).build(), "avg_pool")
 .setOutputs("predictions")
 .setInputTypes(InputType.convolutional(150, 150, 3)) .build(); 

System.out.println(xception.summary());_

**heres my Training phase:**

`TransferLearningHelper transferLearningHelper = new TransferLearningHelper(xception); transferLearningHelper.fitFeaturized(dsi);`

**on training i m getting this error :**

`Exception in thread "main" java.lang.IllegalStateException: Invalid input, does not match configuration: expected [minibatch, numChannels=2048, inputHeight=1, inputWidth=1] but got input array of shape [100, 3, 150, 150]`

**what should i do now …There's no documentation on Xception …i used Xception.summary() function to track my vertices .I have :
output classes=2 
Img input dimension are (150,150 ,3)
Num. of training data =3000
Num of batchSize=150 …guide me please?
I Have also ping this issue on Deeplearning4j forum but NO response .  Suggest Me ?**

**Comment by agibsonccc** [comment]
Closing this. Support questions should be on the forums: https://community.konduit.ai/
